ogc/src/main/scala/geotrellis/server/ogc/wms/WmsParams.scala

```diff
@@ -108,8 +115,7 @@ object WmsParams {
           val height =
             params.validatedParam[Int]("height", { s => Try(s.toInt).toOption })
 
-          val time = params.validatedOptionalParam("time")
-            .map(option => option.map(OgcTimeInterval.fromString))
+          val time = params.validatedTemporalPosition("time")
```

Comment:
It looks like we've lost the ability to support WMS 1.3 queries that use the TimeInterval format since this method only parses results into OgcTimePositions or OgcTimeEmpty.

Querying by list of times and a single time interval look like they should both be supported, no? 

> OGC WMS 1.3.0 specifications (Annex D) **also** allows to display time dimension using a Time list syntax